db/seeds.rb

```diff
@@ -6,13 +6,13 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-employee1 = Employee.where(email: 'ania@o2.pl').first_or_create!(password: 'password')
-employee2 = Employee.where(email: 'marek.z@gmail.com"').first_or_create!(password: 'password')
-employee3 = Employee.where(email: 'ola.gola@hotmail.com').first_or_create!(password: 'password')
-employee4 = Employee.where(email: 'pawel.tkaczyk@wp.pl"').first_or_create!(password: 'password')
-employee5 = Employee.where(email: 'zuza_gala@rv.com').first_or_create!(password: 'password')
+employee1 = Employee.create!(email: 'ania@o2.pl', password: 'okmnji')
```

Comment:
I'd stay with typical seeds-style emails so that others could guess account credentials easily (given you're not working on the project alone)

Reply:
by seeds-style I mean like: `receiver@example.com / password` or `user1@dev.test`etc.

Reply:
Thanks for the hint.